src/plugin.ts

```diff
@@ -4,17 +4,34 @@ import { FastifyPluginAsync } from 'fastify';
 // local
 import { ItemValidationService } from './db-service';
 import { TaskManager } from './task-manager';
-import { itemValidation, itemValidationReview, itemValidationReviews, status } from './schemas';
-import { ItemValidationReview } from './types';
+import {
+  itemValidation,
+  itemValidationGroup,
+  itemValidationProcess,
+  itemValidationReview,
+  itemValidationReviews,
+  status,
+} from './schemas';
+import { GraaspPluginValidationOptions } from './types';
+import { FileTaskManager, ServiceMethod } from 'graasp-plugin-file';
+import { FILE_ITEM_TYPES } from 'graasp-plugin-file-item';
 
-const plugin: FastifyPluginAsync = async (fastify) => {
+const plugin: FastifyPluginAsync<GraaspPluginValidationOptions> = async (fastify, options) => {
   const {
     taskRunner: runner,
     items: { dbService: iS },
   } = fastify;
   const itemValidationService = new ItemValidationService();
   const taskManager = new TaskManager(itemValidationService);
 
+  // classifierApi is the host api of the container running the image classifier
```

Comment:
Would be nicer to add this comment where you define GraaspPluginValidationOptions